HW05-TestAnnotation/src/main/java/ru/otus/TestAnnotation/TestRunner.java

```diff
@@ -0,0 +1,96 @@
+package ru.otus.TestAnnotation;
+
+import ru.otus.TestAnnotation.Annotation.After;
+import ru.otus.TestAnnotation.Annotation.Before;
+import ru.otus.TestAnnotation.Annotation.Test;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TestRunner {
+    public static <T> void run(Class<T> clazz) {
+        List<Method> methodsTest = getTests(clazz);
+        List<Method> methodsAfter = getAfterMethods(clazz);
+        List<Method> methodsBefore = getBeforeMethods(clazz);
```

Comment:
Подумайте как это унифицировать в один метод.
Эти три метода отличаются только типом аннотации.
Т.е. чтобы было примерно так:
```suggestion
        List<Method> methodsTest = getMethods(clazz, ...может что-то надо передать...);
        List<Method> methodsAfter = getMethods(clazz, ...может что-то надо передать...);
        List<Method> methodsBefore = getMethods(clazz, ...может что-то надо передать...);
```